client/apps/edit/components/content/article_layouts/video.jsx

```diff
@@ -0,0 +1,234 @@
+
+    return (
+      <Paragraph
+        html={description || ''}
+        linked
+        onChange={(html) => this.onMediaChange('description', html)}
+        placeholder='Start writing here...'
+      />
+    )
+  }
+
+  editMediaCredits = () => {
+    const { article } = this.props
+    const credits = article.get('media') && article.get('media').credits
+
+    return (
+      <Paragraph
+        html={credits || ''}
+        linked
+        onChange={(html) => this.onMediaChange('credits', html)}
+        placeholder='Start writing here...'
+      />
+    )
+  }
+
+  render () {
+    const { article } = this.props
+    const { uploadProgress } = this.state
+    const media = article.get('media') || {}
+
+    return (
+      <EditVideoContainer>
+        {uploadProgress &&
+          <ProgressBar progress={uploadProgress} />
+        }
+
+        <VideoPreview visible={this.state.showVideo}>
+          <VideoPlayer
+            url={media.url}
+            onLoadedMetadata={this.setDuration}
+          />
+          <div onClick={() => this.toggleVideoPreview(false)}>
+            <IconRemove />
+          </div>
+        </VideoPreview>
+
+        <VideoCover
+          article={article}
+          editTitle={this.editTitle()}
+          editDescription={this.editDescription()}
+          media={media}
+          playVideo={() => this.toggleVideoPreview(true)}
+          hideCover={this.state.showVideo}
+        />
+
+        <EditCoverInput>
+          <FileInput
+            type='simple'
+            onUpload={(src) => this.onMediaChange('cover_image_url', src)}
+            prompt={`+ ${media.cover_image_url ? 'Change' : 'Add'} Cover Image`}
+            onProgress={(uploadProgress) => this.setState({ uploadProgress })}
+          />
+        </EditCoverInput>
+
+        <EditVideoInput>
+          <FileInput
+            video
+            type='simple'
+            sizeLimit={50}
+            onUpload={(src) => this.onMediaChange('url', src)}
+            prompt={`+ ${media.url ? 'Change' : 'Add'} Video`}
+            onProgress={(uploadProgress) => this.setState({ uploadProgress })}
+          />
+        </EditVideoInput>
+
+        <MaxWidthContainer>
+          <VideoAbout
+            article={article}
+            editDescription={this.editMediaDescription()}
+            editCredits={this.editMediaCredits()}
+            color='white'
+          />
+        </MaxWidthContainer>
+
+      </EditVideoContainer>
+    )
+  }
+}
+
+const MaxWidthContainer = styled.div`
+  max-width: 1200px;
+  margin: auto;
+`
+const EditVideoInput = styled.div``
+const EditCoverInput = styled.div``
+
+export const EditVideoContainer = styled.div`
+  position: relative;
+  background-color: black;
+  color: white;
+
+  .ProgressBar {
+    position: fixed;
+    top: 95px;
+    left: 0;
+    right: 0;
+    z-index: 1;
+  }
+
+  ${VideoCoverContainer}, ${VideoAboutContainer} {
+    .public-DraftEditorPlaceholder-root {
+      position: absolute;
```

Comment:
We have to declare `.public-DraftEditorPlaceholder-root` positioning frequently, but it is almost always the same -- it might make sense to move this to a general stylesheet.

Reply:
Is it safe to make this a default? I'm happy to, but I'm not as familiar with the downstream effects of other components that use this.

Reply:
I think it should be fine -- the only place that it is not positioned like above is in the classic and basic titles, where the text is centered.